demos/serverless/README.md

```diff
@@ -28,6 +28,19 @@ npm install
 npm run deploy -- -r us-east-1 -b <my-bucket> -s <my-stack-name> -a meeting
 ```
 
+#### ChimeSDKMeetings Namespace vs Chime NameSpace
+The Chime and the ChimeSDKMeetings are both Amazon Chime's AWS clients to help builders create the Amazon Chime SDK meeting, attendees, enable transcription features and so on. Going forward, we will be using ChimeSDKMeetings client as the default client.
```

Comment:
This sentence doesn't really make sense to me:
* `The Chime`? Maybe mean to phrase it more like '`AWS.Chime` and `AWS.ChimeSDKMeetings` are both AWS SDK clients which can be used to create Amazon Chime SDK meetings, create attendees, and enable optional meeting features such as transcription. `AWS.ChimeSDKMeetings` is intended to replace the previous `AWS.Chime` client.

Reply:
updated